File: 3110-score-of-a-string/3110-score-of-a-string.swift
class Solution {
    func scoreOfString(_ s: String) -> Int {
        var result = 0
        
        var i = 0
        
        while i < s.count - 1 {
            if i + 1 >= s.count {
                break
            }
            
            let a = s[s.index(s.startIndex, offsetBy: i)]
            let b = s[s.index(s.startIndex, offsetBy: i  + 1)]
            let diff = abs(Int(a.asciiValue!) - Int(b.asciiValue!))
            
            result += diff
            
            i += 1
        }
        
        return result
    }
}

File: 3110-score-of-a-string/3110-score-of-a-string.cs
public class Solution
{
    public int ScoreOfString(string s)
    {
        int result = 0;

        for (int i = 0; i < s.Length - 1; i++)
        {
            int c = s[i];
            int n = s[i + 1];

            result += Math.Abs(c - n);
        }

        return result;
    }
}